parsec/mca/pins/task_profiler/pins_task_profiler_module.c

```diff
@@ -252,9 +252,10 @@ task_profiler_exec_count_end(struct parsec_execution_stream_s*   es,
 {
     if (NULL != task->taskpool->profiling_array &&
         task->task_class->task_class_id < task->taskpool->nb_task_classes)
-        PARSEC_TASK_PROF_TRACE(es->es_profile,
-                               task->taskpool->profiling_array[2*task->task_class->task_class_id+1],
-                               task);
+        PARSEC_TASK_PROF_TRACE_FLAGS(es->es_profile,
```

Comment:
This reindentation is incorrect and should not be part of the PR

Reply:
I'm not sure if you meant re-indenting for `task` or reindenting everything according to the beginning of the arguments. My followup will assume you meant the latter.